tables/nl-chardefs.uti.in

```diff
@@ -1,3 +1,102 @@
+dnl Insert a comment in the file generated from this file:
+# ---------------------------------------------------------------------------
+# Beware! This file is automatically generated from nl-chardefs.uti.in. Don't
+# make changes to this file directly.
+# ---------------------------------------------------------------------------
+dnl
+dnl Define some m4 macros to simpify writing the table:
+dnl
+dnl Macro for defining a lowercase letter and corresponding uppercase letter.
+dnl
+dnl     _uplow(<upper>,<lower>,<dots>)
+dnl
+dnl <upper>: uppercase letter
+dnl <lower>: lowercase letter
+dnl <dots>: dot pattern
+dnl
+define(`_uplow',`
+lowercase $2 $3
+uppercase $1 $3f
+base uppercase $1 $2')dnl
+dnl
+dnl Macro for defining a lowercase letter and corresponding uppercase
+dnl letter, to a dot pattern that is already taken by another
+dnl letter. Used for certain accented letters.
+dnl
+dnl     _altuplow(<upper>,<lower>,<dots>,<vdots>)
+dnl
+dnl <upper>: uppercase letter
+dnl <lower>: lowercase letter
+dnl <dots>: dot pattern
+dnl <vdots>: virtual dots to attach (in forward direction only)
+dnl
+define(`_altuplow',`
+noback lowercase $2 $3$4
+noback uppercase $1 $3$4f
+nofor  lowercase $2 $3
+base   uppercase $1 $2')dnl
+dnl
+dnl Macro for defining a superscript character.
+dnl
+dnl     _super(<char>,<dots>,<vdots>)
+dnl
+dnl <char>: character
+dnl <dots>: dot pattern excluding the superscript (346) indicator
+dnl <vdots>: virtual dots to attach (in forward direction only)
+dnl
+define(`_super',`
+noback math $1 346ab-$2$3
+nofor  math $1 346-$2')dnl
+dnl
+dnl Macro for defining a superscript number.
+dnl
+dnl     _supernum(<char>,<dots>)
+dnl
+dnl <char>: character
+dnl <dots>: dot pattern excluding the superscript (346) and numeric (3456) indicators
+dnl
+define(`_supernum',`
+_super($1, 3456-$2, 9a)')dnl
+dnl
+dnl Macro for defining a subscript character.
+dnl
+dnl     _subnum(<char>,<dots>)
```

Comment:
I think this comment is supposed to just say 
``` m4
dnl     _sub(<char>,<dots>)
```

Reply:
Totally right.

Reply:
Fixed